### PR TITLE
Fix Redis namespacing, and add a script to handle the workers.

### PR DESCRIPTION
**Pull request by rotated8**
The script is very slightly modified from [Scholarsphere's.](https://github.com/psu-stewardship/scholarsphere/blob/develop/script/restart_resque.sh)

The namespacing fix comes from projecthydra/sufia@97944c038d22131b1e57a3867c2a5f3a2d859178

### CODE REVIEW

**Comment by rotated8** [comment]
@pmather should review/merge this.

**Comment by rotated8** [comment]
The new script (and other parts of the README) should become part of the bootstrap script(s)

File: scripts/restart_resque.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# Stops and then starts resque-pool in either production or development environment
+# script/restart_resque.sh [production|development]
+
+RESQUE_POOL_PIDFILE="$(pwd)/tmp/pids/resque-pool.pid"
```
Comment (pmather) [inline scripts/restart_resque.sh]: Perhaps we should have a comment in the main comment block stating from where the script should be run?

**Comment by rotated8** [comment]
:+1:

**Pull request merged**